src/System.IO.FileSystem.Primitives/tests/project.json

```diff
@@ -18,9 +18,7 @@
   "supports": {
     "coreFx.Test.netcore50": {},
     "coreFx.Test.netcoreapp1.0": {},
-    "coreFx.Test.net46": {},
-    "coreFx.Test.net461": {},
-    "coreFx.Test.net462": {},
+    "coreFx.Test.netcoreapp1.1": {},
```

Comment:
This is required because netcoreapp1.1 is the default TestTFM now and the project file doesn't change that.